metadata/transportation/road_centerlines

```diff
@@ -8,27 +8,29 @@ a6cf0b81-a99e-4ec3-a9b7-7372ae6a6cbb
-
 ### What is the dataset?
 
+This dataset includes information such as road name, type, speed limit, and infrastructure. To see specifics about the data model including field names, field descriptions, data types and coded value domains, see the [data model spreadsheet](https://docs.google.com/spreadsheets/d/1jQ_JuRIEtzxj60F0FAGmdu5JrFpfYBbSt3YzzCjxpfI/edit#gid=811360546) and the corresponding tabs for update notes. The color-coded section titled "Addressing" is primarily maintained by local stewards, whereas the section titled "DOT/FHWA" is primarily maintained by UDOT. UGRC spatially assigns fields in the section titled "Placename/Area", based on current SGID polygon boundaries.
+
+Please reach out to [our team](https://gis.utah.gov/contact/) with questions concerning this dataset.
+
 ### What is the purpose of the dataset?
 
 The Utah Roads layer plays an important role in [active transportation](https://gis.utah.gov/products/sgid/transportation/) planning in Utah. For example, recent collaborative efforts between UGRC, [UDOT](https://www.udot.utah.gov/connect/,) and [WFRC](https://wfrc.org/) have led to significant enhancements in the bike and pedestrian data not only on this roads layer, but also on the trails and pathways layers. You can contribute to the on-street bike data by [letting us know](https://gis.utah.gov/contact/) where we have gaps or inaccurate information.
 
-As part of UGRC's ongoing effort to standardize schema and improve data shareability, this model has been designed to suggest 'best practices' for stewards. We suggest using this model to estimate how well a given dataset 'translates' to this schema and [Next Generation 911 preparedness](https://www.911.gov/issues/ng911/).
+As part of UGRC's ongoing efforts to standardize schema and improve data shareability, this model has been designed to suggest 'best practices' for stewards. We encourage stewards to use this model to estimate how well a given dataset 'translates' to this schema and [Next Generation 911 preparedness](https://www.911.gov/issues/ng911/).
 
 ### What does the dataset represent?
 
-<!--- How did we decide to delineate the individual polylines? I noticed that for many roads there is more than one polyline segment for one named road. -->
+This dataset contains over 400,000 polyline features representing more than 50,000 roads in Utah. Each polyline represents the centerline of the road and indicates its cardinal direction. Divided highways such as I-15 include both a positive and negative direction segment.
```

Comment:
This is the new description for what the dataset represents. I will be the first to admit that I feel as though this description reveals my limited knowledge in this area. Do let me know what else you think I could add to it to improve it!
@gregbunce @steveoh

Reply:
#thinking

Reply:
what about something like this: 

> Utah Roads is a comprehensive statewide roads dataset for mapping, addressing, and route planning. This data follows national standards for [NENA NG911](https://www.nena.org/page/standards) and [USDOT ARNOLD](https://highways.dot.gov/safety/data-analysis-tools/rsdp/rsdp-tools/arnold-reference-manual) and is updated monthly for key areas, with additional counties updated throughout the year. UGRC collaborates with various agencies to ensure data accuracy and consistency.

Reply:
I think that's a lovely summary paragraph, but I don't think it quite belongs in the section "What does this dataset represent". Thus far, I have been using this heading to discuss what the points, lines, or polygons actually mean--hence, what the visual data itself represents. The information in the suggested paragraph is already found throughout the rest of the metadata, thus I think it would be best to put this paragraph up at the top under "Summary". 

So far, I have been attempting to fill in every heading from the template with information, but it occurs to me that perhaps people don't need to know what the lines mean? I mean, it is fairly self-explanatory that the lines are where the roads are and contain associated attributes. So perhaps we don't need anything under "what does this dataset represent" at all. Thoughts?

Reply:
That looks awesome to me! Thank you for working with me on this!